### ISSUE TITLE
Loot crates + Scrapping 

### ISSUE THREAD

**Issue by DebugMC** [Wynncraft/Issues#7255]
You can not scrap 5 godly rewards, it says you do not have 5 of the tier. This is not true though, I put in five of the same tier normally it gives me a higher tier. Godly should prob give me godly. @AscendedKitten closed my original issue calling it a feature request, but its not. As stated on that post I asked a mod (JuicedBananas) a while ago and they told me it should be possible. 

Also if you activate a disguise, weapon skin, or attack effect then scrap them, they are not deactivated. (Disguises on relog, but the rest don´t reset)

Original post: #7246

**Comment by AscendedKitten** [comment]
In my opinion it's more of a problem with wording, rather than an actual issue / bug, but I'll leave it open.
@Jumla is probably the only one who can "fix" it though.

**Comment by DebugMC** [comment]
Right, if it wasn't meant to be done, it should say 'you can not scrap 5 of this tier' or something. Right now it just says you haven't put enough items in even though I have filled the GUI

**Comment by DebugMC** [comment]
...how is this "(Wording)" xD

**Comment by AscendedKitten** [comment]
I am quite sure it wasn't intended as a feature, but you are right, the message should be different.

**Comment by DebugMC** [comment]
Perhaps 'you can not scrap 5 godlies" or something... 

I love how google says godlies isn't a word with no spelling suggestions kappa

**Comment by Moe-Ronickah** [comment]
It says what it says, because there was no exception made in the code, apparently, for g/dly items.  This isn't a big deal, and, as AscendedKitten said, it's up to Jumla to fix it.  We don't really need this left open, even though it's not fixed, yet, because it's so minor of an issue, but it can stand here, so other players can find it.

**Comment by DebugMC** [comment]
Allright.
